tests/notifications/test_api.py

```diff
@@ -0,0 +1,128 @@
+import json
+
+from django.core.urlresolvers import reverse
+
+from bulbs.notifications.models import Notification
+from bulbs.utils.test import BaseAPITestCase
+
+
+DEFAULT_NOTIFICATION_DATA = {
+    'internal_title': 'rio: part 1.',
+    'headline': 'You can do it nicky!',
+    'body': 'Little Nicky is the greatest movie of all time.',
+    'image': {'id': 1},
+    'clickthrough_url': 'https://www.clickhole.com'
```

Comment:
handled in another test so I could check 
- without config & without stored
- with config & without stored
- without config & with stored